ref/String_Operators.md

```diff
@@ -194,6 +194,29 @@ defines the function `f(x)` to be `sin(x)+cos(x)`.
 
 ------
 
+
+#### Parsing a CSV string: `parseCSV(‹string›)`
+
+**Description:**
+This operator parses a CVS string to an list of lists.
+
+    - skip test: parseCSV module not linked in for unit tests.
+    > parseCSV("Foo,1.0
+    > Bar,2.3
+    > abc,5.0
+    > asd,1.3
+    > sdf,4.3
+    > lala,2.3
+    > Barrr,1.7
+    > Baz,4.7");
+    < [[Foo, 1], [Bar, 2.3], [abc, 5], [asd, 1.3], [sdf, 4.3], [lala, 2.3], [Barrr, 1.7], [Baz, 4.7]]
+
+Numbers and Booleans are converted to their respective CindyScript counterparts, if this is not the desired behavior use the `autoconvert` modifier and set it to `false`.
+The modifiers `delimiter` and `newline` can be used to set the separating and linebreak characters.
```

Comment:
What are the defaults? [RFC 4180](https://tools.ietf.org/html/rfc4180) uses CRLF but points out that “implementors should be aware that some implementations may use other values”. Common sense tells me that while *parsing* CSV it would make sense to allow all CRLF, CR and LF as end of line terminators (except inside a quoted string).